agent/config.go

```diff
@@ -27,8 +27,17 @@ type Config struct {
 	// NotifyNodeChange channel receives new node changes from session messages.
 	NotifyNodeChange chan<- *api.Node
 
+	// NotifyTLSChange channel sends new TLS information changes, which can cause a session to restart
+	NotifyTLSChange <-chan *api.NodeTLSInfo
+
+	// NotifyRootChange channel receives new trust root certificate changes from session assignments
+	NotifyRootChange chan<- []byte
```

Comment:
Now I do see it. Maybe it wasn't in the version I was looking at a few seconds ago.